packages/runtime/src/context.ts

```diff
@@ -36,12 +38,17 @@ export type RuntimeCallback<
 /**
  * Create a new runtime context from an environment.
  */
-export function createContext(environment: IntegrationEnvironment): RuntimeContext {
+export function createContext(
+    environment: IntegrationEnvironment,
+    event: FetchEvent
+): RuntimeContext {
     return {
         environment,
         api: new GitBookAPI({
             endpoint: environment.apiEndpoint,
             authToken: environment.authToken,
         }),
+
+        event,
```

Comment:
We need to discuss with team if this is the best way to expose it.

Reply:
reverting this for now. replacing with a keepalive on the fetch to test.